templates/actors/parts/actor-spellbook.hbs

```diff
@@ -30,91 +30,92 @@
 
 <ol class="items-list inventory-list">
 {{#each spellbook as |section|}}
-    <li class="items-header spellbook-header flexrow">
-        <div class="item-name flexrow">
-            <h3>{{section.label}}</h3>
-            <div class="spell-slots">
-                {{#if section.usesSlots}}
-                <input type="text" name="system.spells.{{section.prop}}.value" value="{{section.uses}}" placeholder="0"
-                       data-dtype="Number">
-                <span class="sep"> / </span>
-                <span class="spell-max" data-level="{{section.prop}}" data-slots="{{section.slots}}">
-                    {{{section.slots}}}
-                    {{#if section.editable}}
-                    <a class="slot-max-override" data-tooltip="DND5E.SpellProgOverride">
-                        <i class="fas fa-edit"></i>
-                    </a>
+    <div class="spellbook-section" {{#each section.dataset as |v k|}}data-{{k}}="{{v}}"{{/each}}>
```

Comment:
`<div>` is not actually a valid child of an `<ol>` element, but neither is another `<ol>`, which is something we also already do here. I think the 'strictly correct' markup we should use here (and in `actor-inventory.hbs`) is something like:
```html
<ol class="items-list inventory-list">
  <li class="spellbook-section">
    <header class="items-header spellbooks-header flexrow"><!-- ... --></header>
    <ol class="item-list"><!-- ... --></ol>
  </li>
  <!-- ... -->
</ol>
```

That's probably too disruptive of a change to make as part of this PR, but we could keep the markup mostly the same as before at the expense of some slightly more complicated logic in `_onDropSpell`:

```hbs
<!-- actor-spellbook.hbs -->
<li class="items-header spellbook-header flexrow" {{#each section.dataset as |v k|}}data-{{k}}="{{v}}"{{/each}}>
```

```js
// _onDropSpell
let header = this._event.currentTarget.closest(".items-header"); // Dropped directly on the header.
if ( !header ) {
  const list = this._event.currentTarget.closest(".items-list"); // Dropped inside an existing list.
  header = list?.previousElementSibling;
}
const mode = header?.dataset ?? {};
```

Let me know what you think.

Reply:
> Let me know what you think.

My (very blunt) opinion is that don't see this as *too* important, right now at least, and that the actor sheets are due for a revisit overall. I think it could be good to see to this part in particular at such a time.